src/fsverity/ioctl.rs

```diff
@@ -53,7 +69,10 @@ pub struct FsVerityDigest<F> {
 // #define FS_IOC_MEASURE_VERITY   _IORW('f', 134, struct fsverity_digest)
 type FsIocMeasureVerity = ioctl::ReadWriteOpcode<b'f', 134, FsVerityDigest<()>>;
 
-pub fn fs_ioc_measure_verity<F: AsFd, H: FsVerityHashValue>(fd: F) -> Result<H> {
+/// Measure the fsverity digest of the provided file descriptor.
+pub fn fs_ioc_measure_verity<F: AsFd, H: FsVerityHashValue>(
```

Comment:
Tangential but since this function is much less of a trivial wrapper for the ioctl, I am less certain about having the module named `ioctl` (or less invasively perhaps we could rename this function to just `measure_verity` or something? anyways...)

Reply:
Not so tangential.  It's actually a good point.

I think it would make sense to add the compare/error stuff maybe at the toplevel `fsverity/mod.rs`?

Reply:
Oh.  That's exactly what you did.  Thanks!